game-app/game-core/src/main/java/games/strategy/triplea/ui/UnitChooser.java

```diff
@@ -393,30 +427,28 @@ void disableMax() {
     total = -1;
   }
 
-  private static final class ChooserEntry {
-    private static final Insets nullInsets = new Insets(0, 0, 0, 0);
+  /**
```

Comment:
Please check this javadoc and look to rephrase, possibly stating more simply and/or more directly  (lines 433~435 do not make a lot of sense)

Reply:
Next commit will contain another try. Documenting all this subclass by subclass seems quite suboptimal to me. A comprehensive design document would make more sense. But I did not write UnitChooser etc. I only started out with a bugfix that allowed me continuing to play "A game of ice and fire". I am starting to feel nipped by suboptimal code quality algorithms. Only because those queried me to write a piece of javadoc here, it did this

Reply:
I'm sympathetic. There is a lot of practice to be had here working with a legacy code base.